### PR TITLE
Bump psammead packages to latest

### LINKED ISSUE #4402: Update packages for event tracking
**Is your feature request related to a problem? Please describe.**
We made changes to several components to enable passing props for event tracking and these components should be bumped in Simorgh. Make changes required to not break the components.

**Describe the solution you'd like**
Bump all the components updated in https://github.com/bbc/psammead/pull/2377/files#diff-649fabcf7d0f0c3dc0bfa5b342f106b2 and make changes were necessary.

**Describe alternatives you've considered**
Not bump and block PRs like https://github.com/bbc/simorgh/pull/4382

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

### PR DESCRIPTION
**Pull request by ghost**
Resolves #4402

**Overall change:** _Bump psammead packages to their latest stable versions._

**Code changes:**
Updated packages.
```
@bbc/gel-foundations
@bbc/psammead-assets
@bbc/psammead-brand
@bbc/psammead-calendars
@bbc/psammead-caption
@bbc/psammead-consent-banner
@bbc/psammead-copyright
@bbc/psammead-figure
@bbc/psammead-headings
@bbc/psammead-image
@bbc/psammead-image-placeholder
@bbc/psammead-inline-link
@bbc/psammead-media-indicator
@bbc/psammead-navigation
@bbc/psammead-paragraph
@bbc/psammead-section-label
@bbc/psammead-sitewide-links
@bbc/psammead-story-promo-list
@bbc/psammead-styles
@bbc/psammead-timestamp-container
@bbc/psammead-useful-links
@bbc/psammead-visually-hidden-text
```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by amywalkerdev** [comment]
This will need manual testing as we're going to new major versions on a few of the packages

**Review by AlistairGempf — APPROVED** [review]
Looks good to me

**Comment by AlistairGempf** [comment]
I think this may end up conflicting with #4390.

**Comment by ghost** [comment]
@AlistairGempf Thanks for the heads-up, I didn't see the PR. I will revert the changes made to storybook.

**Comment by thekp** [comment]
@sadick254 

from this morning, dependabot has created a few more bumping PRs. could you merge them into this PR please: 
https://github.com/bbc/psammead/pulls?q=is%3Apr+is%3Aopen+label%3Adependencies

**Review by RayNjeri — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]
Awesome. This fixes a duplicate package error I'm having, which prevents https://github.com/bbc/simorgh/pull/4388 from building.

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]
This needs manual testing, due to the major version bumps.

File: src/app/containers/FrontPageSection/index.jsx [hunk 1]
```diff
@@ -155,6 +155,7 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
         dir={dir}
         linkText={isLink ? seeAll : null}
         href={href}
+        data-section-divider={sectionLabelId}
```
Comment (amywalkerdev) [inline src/app/containers/FrontPageSection/index.jsx]: This needs to be removed

File: src/app/containers/Navigation/index.jsx [hunk 2]
```diff
@@ -36,6 +36,7 @@ const NavigationContainer = () => {
               active={active}
               currentPageText={currentPage}
               service={service}
+              data-navigation={`${title}_${url}`}
```
Comment (amywalkerdev) [inline src/app/containers/Navigation/index.jsx]: And this

**Review by amywalkerdev — CHANGES_REQUESTED** [review]

File: src/app/containers/FrontPageSection/index.jsx (hunk 1 above)
Comment (ghost) [reply src/app/containers/FrontPageSection/index.jsx]: Done.

**Review by amywalkerdev — APPROVED** [review]
Looks good now, thanks 👍

**Comment by PriyaKR** [comment]
@jamesbrumpton and me have tested this together and looks good.Can be merged.

**Pull request merged**